### PR TITLE
Removes clockcult from the game entirely.

### PR DESCRIPTION
**Pull request by Iamgoofball**
# WHY
No one likes it. It's boring, it's obnoxiously complex, it's filled with loads of shitty fucking shoehorned edgelord lore including it's own language and translator like some kind of shitty LOTR fanfiction.

We have too many shit conversion modes. Axing this and Gang will help.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
today I learned: shuttle walls exist only by accident due to being defined in a proc

**Comment by CPTANT** [comment]
Is it remove gamemodes week?

**Comment by Iamgoofball** [comment]
@Xhuis if your rework gets merged, I'll close this

if Joan closes your rework and tries to steal it again this will stay open.

**Comment by Ninja137** [comment]
But Clockcult is actually a nice change from the typical "Run around and just murder everyone because literally every gamemode revolves around just arming up and TDMing people until you win", since it actually includes different gameplay elements like resource management, structure building, and actually requires teamwork.

**Comment by Ratvar** [comment]
Ninja127, he talks about oldclock, not newcult Xcuis rework which has all pluses you show and no tediusness of oldclock.

**Comment by Ninja137** [comment]
If by newclock you mean that fucking abomination where you sit at Reebe for an hour and do literally nothing, with zero risk, no, fuck off, that's goddamn cancer.

**Comment by evsey9** [comment]
"Noone likes it" Speak for yourself.
"It's boring" It's not.
"It's obnoxiously complex" Lets have at least fuckin SOMETHING that is complex in this game, for fucks sake.
"it's filled with loads of shitty fucking shoehorned edgelord lore including it's own language and translator like some kind of shitty LOTR fanfiction." Blood cult has it's own language too. Elaborate on the translator part.

**Comment by InsaneHyena** [comment]
BUT MUH CODEBABY

File: code/game/turfs/simulated/wall/misc_walls.dm [hunk 1]
```diff
@@ -68,3 +68,7 @@
 	desc = "A huge chunk of rusted reinforced metal."
 	icon = 'icons/turf/walls/rusty_reinforced_wall.dmi'
 	hardness = 15
+
+/turf/closed/wall/shuttle
```
Comment (ChangelingRain) [inline code/game/turfs/simulated/wall/misc_walls.dm]: Should not actually exist, remove properly instead.

**Comment by Cyberboss** [comment]
How many times in a year is this gonna get PR'd?

**Comment by PopNotes** [comment]
someone should make a PR that auto-denies all future goofball PRs

**Comment by ma44** [comment]
But then we can't have dorfs :-1:

**Comment by kilkun** [comment]
Clock cult fucking sucks

Come back when you actually have to be good at the game and combat to succeed as clock cult, instead of just spamming turrets and aoe magic until you win.

File: code/game/turfs/simulated/wall/misc_walls.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/turfs/simulated/wall/misc_walls.dm]: Negative, maps use it.

**Comment by Xhuis** [comment]
The rework is always being tweaked and changed, and I'm taking it slowly so as to fine-tune issues as they come up. I don't expect it to be merged for a while.

File: code/game/turfs/simulated/wall/misc_walls.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/turfs/simulated/wall/misc_walls.dm]: they shouldn't

Comment (ChangelingRain) [inline code/game/turfs/simulated/wall/misc_walls.dm]: since it has NO STUFF and is a DEFAULT WALL

**Comment by kilkun** [comment]
xhuis please you are the only hope to make the gamemode bearable

**Comment by FantasticFwoosh** [comment]
Im split halfways in agreeing with goof and defending clockies on the basis of a incomplete arguement vs them

- Its not clear what iteration of clockcult goof is targeting, Xhuis has been co-operative with user feedback and given a chance can deliver a game-mode that is workable, unless the concept is critically flawed.

- Critically the gameplay of (**old**)clockcult however is flawed because base building is so tacky and cheesable with tower defense courses & popup maint bases, its repeating the mistakes of bloodcult before robustin changed it where crewmembers can search maint with mesons to find bases.
________________________________________________
Remember we removed buildings from bloodcult because they kept on being lynched, and now the reebe is not griefproof from naturally strong creatures like 🦍 and hulks that can wreak major destruction.

IE see presented evidence : #21755 and #21723 because critically if you break the clock-cultists shit and defeat its members (gorilla's can't be affected by spear throws) the clock cult WILL be damaged by it mechanically compared to blood cult which is more loose and focuses less on building.

**Comment by kilkun** [comment]
actually old clock cult sucked because you could just BoH bomb the gateway and they had no recourse.

**Comment by ChangelingRain** [comment]
We uh, kind of didn't remove buildings from bloodcult?

**Comment by Bawhoppen** [comment]
@Iamgoofball I swore I purged those damn walls

File: code/game/turfs/simulated/wall/misc_walls.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/game/turfs/simulated/wall/misc_walls.dm]: fuck

Comment (Bawhoppen) [reply code/game/turfs/simulated/wall/misc_walls.dm]: yes it shouldn't exist, get it out

**Pull request closed without merging**